xwiki-commons-core/xwiki-commons-extension/xwiki-commons-extension-api/src/main/java/org/xwiki/extension/Extension.java

```diff
@@ -260,7 +260,7 @@
      */
     default ExtensionId getExtensionFeature(String featureId)
     {
-        return getFeatures().contains(featureId) ? new ExtensionId(featureId, getId().getVersion()) : null;
+        return getExtensionFeatures().contains(featureId) ? new ExtensionId(featureId, getId().getVersion()) : null;
```

Comment:
This does not work. getFeatures() and getExtensionFeatures() don't return the same thing.